core/indexer/common_test.go

```diff
@@ -23,7 +25,11 @@ func TestGetTransactionByType_SC(t *testing.T) {
 	header := &block.Header{Nonce: 2}
 
 	resultTx := getTransactionByType(smartContract, txHash, mbHash, blockHash, mb, header, "")
-	require.Equal(t, nonce, resultTx.Nonce)
+	expectedTx := &Transaction{
+		Hash: hex.EncodeToString(txHash), MBHash: hex.EncodeToString(mbHash), BlockHash: hex.EncodeToString(blockHash),
```

Comment:
This should be formatted: one property under another

Reply:
done.